pkg/controllers/hub/trafficmanagerprofile/controller.go

```diff
@@ -271,6 +280,7 @@ func generateAzureTrafficManagerProfile(profile *fleetnetv1alpha1.TrafficManager
 		Properties: &armtrafficmanager.ProfileProperties{
 			DNSConfig: &armtrafficmanager.DNSConfig{
 				RelativeName: ptr.To(fmt.Sprintf(DNSRelativeNameFormat, profile.Namespace, profile.Name)),
+				TTL:          ptr.To(DefaultDNSTTL), // no default value on the server side, using 60s same as portal's default config
```

Comment:
This is not configurable from current API right?

Reply:
Yes, we don't allow cx to config for now.